.vscode/extensions.json

```diff
@@ -1,10 +0,0 @@
-{
-    // See http://go.microsoft.com/fwlink/?LinkId=827846
-    // for the documentation about the extensions.json format
-    "recommendations": [
-        "platformio.platformio-ide"
-    ],
-    "unwantedRecommendations": [
```

Comment:
Why is unwanted cut out. I assume platformio is removed because the instructions later call out for Ardunino IDE.